### PR TITLE
Update next.config.js

### PR DESCRIPTION
**Pull request by LorenaCapraru**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+  images: {
+    domains: ['avatars.githubusercontent.com'],
+}
```
Comment (ehwus) [inline next.config.js:5]: Can you sort the formatting here? It's a bit jarring!

```
const nextConfig = {
  images: {
    domains: ['avatars.githubusercontent.com'],
  }
}

**Review by ehwus — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 1 above)
Comment (LorenaCapraru) [reply next.config.js]: done, I'll pay more attentions to formatting.

**Review by ehwus — APPROVED** [review]

**Review by nataliiazab — APPROVED** [review]
thanks Lorena!

**Review by momahboobian — APPROVED** [review]
Thank you so much for amendmet.

**Pull request merged**